besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -3312,6 +3320,39 @@ private void setMergeConfigOptions() {
             .isPresent());
   }
 
+  private void validateNearHearCheckPointSyncRequirements() {
```

Comment:
Given BesuCommand is such a huge class, is it possible to split this and the tests out into its own class?